tests/post_training/model_scope.py

```diff
@@ -76,6 +76,17 @@
         "backends": [BackendType.FX_TORCH, BackendType.TORCH, BackendType.CUDA_TORCH, BackendType.OV, BackendType.ONNX],
         "batch_size": 128,
     },
+    {
+        "reported_name": "torchvision/mobilenet_v3_small_BC",
```

Comment:
What differ between timm and torchvison mobilenetv3?
Can be timm model be repalaced to torchvision as resnet18?